bedrock/firefox/templates/firefox/new.html

```diff
@@ -150,9 +158,36 @@ <h1 class="large" id="main-headline">
       <div id="newsletter-latest">
         {{ email_newsletter_form() }}
       </div>
-    </div>{#-- /#download-ready --#}
+    </div>{#-- /#scene1-main --#}
+
+    <div id="scene2-main" class="scene-content scene2 hidden">
+    {% if l10n_has_tag('firefox_new_mobile_promo') and waffle.switch('firefox-ios-global') %}
+      <p class="help">
+        {{_('Your download should begin automatically. If not, <a id="%s" href="">click here</a>. It could take a few minutes, but it’s worth the wait.')|format('direct-download-link')}}
+        <a rel="external" href="https://support.mozilla.org/products/firefox?utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=need-help-link">{{_('Need help?')}}</a>
+      </p>
+
+      <div id="thankyou" tabindex="0">
+        {{ _('While it installs, add it to your Android and iOS devices.') }}
+      </div>
 
-    <div id="download-started" class="download-content hidden">
+      <div class="mobile-download-buttons">
+        <ul>
+          <li class="android">
+            <a rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
+              {{ high_res_img('firefox/android/btn-google-play.png', {'alt': _('Get it on Google Play'), 'width': '129', 'height': '45', 'l10n': True}) }}
+            </a>
+          </li>
+          <li class="ios">
+            <a href="{{ settings.APPLE_APPSTORE_FIREFOX_LINK }}&amp;ct=mozorg-scene_2-appstore-button">
+              <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+            </a>
+          </li>
+        </ul>
+      </div>
```

Comment:
So here is a fun fact about the above:
The scene switches once download is clicked in all browsers but:

1) IE6 - There are no clickable areas nor badges/buttons
2) IE7/8 - The Android badge is shown but not the iOS one (SVG)